src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+            'Security'                            : 'Sécurité',
+            'Trash'                               : 'Corbeille',
+            'Details'                             : 'Détails',
+            'Edit'                                : 'Modifier',
+            'Rename'                              : 'Renommer',
+            'Delete'                              : 'Effacer',
+            'Restore'                             : 'Restaurer',
+            'New Password'                        : 'Nouveau mots de passe',
+            'New Folder'                          : 'Nouveau dossier',
+            'New Tag'                             : 'Nouveau mot clé',
+            'Secure'                              : 'Sécurisé',
+            'Weak'                                : 'Faible',
+            'Weak (Duplicate)'                    : 'Faible (Dupliqué)',
+            'Weak (Outdated)'                     : 'Faible (Obsolète)',
+            'Breached'                            : 'Compromis',
+            'Notes'                               : 'Notes',
+            'Share'                               : 'Partager',
+            'Revisions'                           : 'Révisions',
+            'QR Code'                             : 'QR Code',
+            'Password'                            : 'Mot de passe',
+            'Username'                            : 'Nom d’utilisateur',
+            'Website'                             : 'Site internet',
+            'Add Tags...'                         : 'Ajouter des mots clés…',
+            'Color'                               : 'Couleur',
+            'Name'                                : 'Nom',
+            'Label'                               : 'Étiquette',
+            'General'                             : 'Général',
+            'Properties'                          : 'Propriétés',
+            'Numbers'                             : 'Nombre',
+            'Special Characters'                  : 'Caractères spéciaux',
+            'More Options'                        : 'Plus d’options',
+            'Favorite'                            : 'Favori',
+            'Encryption'                          : 'Chiffrement',
+            'On the server'                       : 'Sur le serveur',
+            'Custom Fields'                       : 'Champs Personnalisés',
+            'Secret'                              : 'Secret',
+            'File'                                : 'Fichier',
+            'Value'                               : 'Valeur',
+            'Toggle visibility'                   : 'Changer la visibilité',
+            'Generate password'                   : 'Générer un mot de passe',
+            'Simple Server Side Encryption V1'    : 'Version 1 simple chiffrement coté serveur',
+            'Take some notes'                     : 'Prendre quelques notes',
+            'Save'                                : 'Sauver',
```

Comment:
`Sauver` has a different meaning, this should be `Sauvegarder`.

Reply:
i will correct this